utils/augmentations.py

```diff
@@ -382,7 +401,14 @@ def __call__(self, im):  # im = np.array HWC
         imh, imw = im.shape[:2]
         m = min(imh, imw)  # min dimension
         top, left = (imh - m) // 2, (imw - m) // 2
-        return cv2.resize(im[top:top + m, left:left + m], (self.w, self.h), interpolation=cv2.INTER_LINEAR)
+        # Use torchvision transforms to have parity with production
+        cropped_im = torch.from_numpy(im[top:top + m, left:left + m].transpose(2, 0, 1))
+        resized_im = TF.resize(
+            img=cropped_im,
+            size=[self.h, self.w],
+            interpolation=TF.InterpolationMode.BILINEAR,
+        )
+        return resized_im.numpy().transpose(1, 2, 0)
```

Comment:
Could we confirm this is the correct colorspace?

Reply:
I believe the colorspace is dependent on the hyperparameters you select

Reply:
This is converting HWC to CHW input